contact.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+
+        <title>Contacting Staff</title>
+    </head>
+    <body>
+        <h1>How to contact server staff?</h1>
+
+        <p>If there is an isse that requires staff attention, please do not ping or direct message a staff member to report it as this is against server rules and may 
+            result in a warn. Instead please use the ModMail bot with the method below to bring any issues to the mod team's attention.
+        </p>
+
+        <h2>The ModMail Bot:</h2>
+        <p>To report any issues you can you the ModMail bot to open a ticket with server staff.</p>
+
+        <ol>
+            <li>
+                <h3>Send a message containing your issue to the bot</h3>
+                <img src="src/img/modmail0.png">
+                <img src="src/img/ModMail1.png">
+            </li>
+            <li>
+                <h3>Select the Codecademy Community server by reacting with the appropriate number</h3>
+                <p>(You may have to use the arrow reactions to scroll if you are in a large number of servers)</p>
+            </li>
+            <li>
+                <h3>Sent!</h3>
+                <p>Your message has now been sent to server staff.</p>
+                <img src="src/img/ModMail2.png">
+            </li>
+        </ol>
+
+        <h2>If the ModMail bot is offline:</h2>
+        <p> If the ModMail bot is offline and you need to report a serious violation of server rules, you can ping an online Admin or Moderator to bring our attention to the violation. However, please reserve this only for when the ModMail bot is down or if the violation requires immediate attention.</p>
```

Comment:
I think the "if the violation requires immediate attention" part should be placed in the `How to contact server staff?` section. You can also include an example, like scam links and mentions being spammed in all the channels.